numpy/lib/stride_tricks.py

```diff
@@ -86,6 +86,7 @@ def as_strided(x, shape=None, strides=None, subok=False, writeable=True):
     Vectorized write operations on such arrays will typically be
     unpredictable. They may even give different results for small, large,
     or transposed arrays.
+
```

Comment:
Not sure where this came from, but it's probably a good idea. Will remove if there are objections.